### PR TITLE
SDK doc fixes

### PR DESCRIPTION
**Pull request by kevinheavey**
#### Problem
Fix some `cargo doc` warnings in SDK crates

### CODE REVIEW

File: sdk/decode-error/src/lib.rs [hunk 1]
```diff
@@ -18,8 +18,8 @@ use num_traits::FromPrimitive;
 /// `FromPrimitive`, it is only used correctly when the caller is certain of the
 /// original error type.
 ///
-/// [`ProgramError`]: crate::program_error::ProgramError
-/// [`ProgramError::Custom`]: crate::program_error::ProgramError::Custom
+/// [`ProgramError`]: https://docs.rs/solana-program-error/latest/solana_program_error/enum.ProgramError.html#variant.Custom
```
Comment (joncinque) [inline sdk/decode-error/src/lib.rs]: Is this correct?
```suggestion
/// [`ProgramError`]: https://docs.rs/solana-program-error/latest/solana_program_error/enum.ProgramError.html
```

**Review by joncinque — COMMENTED** [review]
Just one little nit, thanks for cleaning up

File: sdk/decode-error/src/lib.rs (hunk 1 above)
Comment (kevinheavey) [reply sdk/decode-error/src/lib.rs]: fixed

**Review by joncinque — APPROVED** [review]
Thanks!

**Pull request merged**